src/views/registration-js/attendees/fields/birth.php

```diff
@@ -33,47 +33,52 @@
 		<label
 			class="tribe-common-b2--min-medium tribe-tickets-meta-label"
 			for="<?php echo esc_attr( $option_id ); ?>"
-		><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+		>
+			<?php echo wp_kses_post( $field['label'] ) . ' ' . tribe_required_label( $required ); ?>
```

Comment:
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'tribe_required_label'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 61d78444e4aa51de65feb327f9893cab</sub>